packages/iov-ethereum/src/ethereumconnection.ts

```diff
@@ -740,14 +764,18 @@ export class EthereumConnection implements AtomicSwapConnection {
     return { ...transaction, fee: await this.getFeeQuote(transaction) };
   }
 
-  public async getSwaps(query: AtomicSwapQuery): Promise<ReadonlyArray<AtomicSwap>> {
+  public async getSwaps(
+    query: AtomicSwapQuery,
+    minHeight: number = 0,
+    maxHeight: number = Number.MAX_SAFE_INTEGER,
```

Comment:
interesting compatible addition

Reply:
Should I update the type? And if so, would it be better to pass it an options object so we could specify a time for relevant blockchains instead?

Reply:
If this turns out to be useful long term, the better place to put them is inside of `AtomicSwapQuery`, just like in `TransactionQuery`. For now this looks fine to me.